pages/common/mutool.md

```diff
@@ -1,28 +1,28 @@
 # mutool
 
 > Convert PDF files, query information and extract data.
-> More information: <https://mupdf.com/docs>.
+> More information: <https://mupdf.readthedocs.io/en/latest/mupdf-command-line.html>.
 
 - Convert pages 1-10 into 10 PNGs:
 
-`mutool convert -o {{image%d.png}} {{file.pdf}} {{1-10}}`
+`mutool convert -o {{path/to/output.png}} {{path/to/input.pdf}} {{1-10}}`
```

Comment:
I'm pretty sure that doesn't work with the output file name not containing a wildcard

Reply:
I will try this command, and get back to your suggestion.

Reply:
Tried it, now the updated suggestion is

```suggestion
`mutool convert -o {{path/to/output%nd.png}} {{path/to/input.pdf}} {{1-10}}`
```
The command works without `%d` too creating output in `1,2,3,4` sequence which is similar to adding it, when you add `%2d` then the sequence is like `01,02,03,04`, maybe we can update it to `nd` and highlight n in description.